README.md

```diff
@@ -122,7 +122,7 @@ Celerity's runtime behavior:
   automatically assign a unique device to each worker on a host.
 - `CELERITY_PROFILE_KERNEL` controls whether SYCL queue profiling information
   should be queried (currently not supported when using hipSYCL).
-- `CELERITY_GRAPH_PRINT_MAX_VERTS` sets the maximum number of vertices the
-  task/command graphs can have above which their GraphViz output will be omitted.
+- `CELERITY_RECORDING` enables recording of the generated tasks and commands,
```

Comment:
I thought about this, but to me `CELERITY_RECORDING` is a bit more intuitive in this use case, since it makes it clearer (at least in my mind) that I'm just toggling something on and off. `CELERITY_RECORD` could also specify e.g. what to record, or where to record it.